internal/daysteps/daysteps.go

```diff
@@ -1,23 +1,62 @@
 package daysteps
 
 import (
+	"errors"
+	"fmt"
+	"strconv"
+	"strings"
 	"time"
+
+	"github.com/Yandex-Practicum/go1fl-4-sprint-final/internal/spentcalories"
 )
 
 var (
-	StepLength = 0.65 // длина шага в метрах
+	StepLength       = 0.65                                // длина шага в метрах
+	ErrInvalidFormat = errors.New("Неверный формат ввода") // Обработка ошибки
 )
 
+// parsePackage возвращает данные дневной активности(Шаги за прогулку и продолжительность прогулки)
 func parsePackage(data string) (int, time.Duration, error) {
-	// ваш код ниже
+	str := strings.Split(data, ",")
+	if len(str) != 2 {
+		return 0, 0, ErrInvalidFormat
+	}
+
+	steps, err := strconv.Atoi(str[0])
+
+	if err != nil {
+		return 0, 0, ErrInvalidFormat
```

Comment:
1. Теряешь сообщение об ошибки
2. Ошибку лучше писать на английском
3. Ошибку лучше начинать с маленькой буквы
```suggestion
		return 0, 0, fmt.Errorf("conversion error: %w", err)
```